Move macpaw-gemini into cask-versions (1.5.17,1461238766)

toonetown opened the pull request:
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-versions/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-versions/issues) where that cask was already refused.
- [x] Checked the cask follows the requirements of [acceptable casks](https://github.com/caskroom/homebrew-versions#acceptable-casks).
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

adidalal commented:
This should be called `gemini-classic` now

Homepage is http://macpaw.com/gemini-classic

toonetown commented:
OK - I've renamed it - and updated the home page, but I left the url/appcast stanzas the same, because that is where they actually are downloaded from.  https://dl.devmate.com/com.macpaw.site.GeminiClassic/GeminiClassic.dmg is just a shortcut to whatever the "latest" version is.
